### PR TITLE
Add text chat functionality

### PR DESCRIPTION
**Pull request by lalalune**

### CODE REVIEW

File: wsrtc.js [hunk 1]
```diff
@@ -4,13 +4,24 @@ import {ensureAudioContext, getAudioContext} from './ws-audio-context.js';
 import {encodeMessage, encodeAudioMessage, encodePoseMessage, encodeTypedMessage, decodeTypedMessage, getEncodedAudioChunkBuffer, getAudioDataBuffer/*, loadState*/} from './ws-util.js';
 import * as Z from 'zjs';
 
+import metaversefileApi from 'metaversefile';
```
Comment (avaer) [inline wsrtc.js:7]: Don't use `metaversefile` here. wsrtc should be more independent.
Emit the message/data you need from the connection socket, and then keep the player logic in `app`.

File: wsrtc.js [hunk 2]
```diff
@@ -164,7 +176,28 @@ class WSRTC extends EventTarget {
...
+      const _handleChatMessage = (e, dataView) => {
+        const textDecoder = new TextDecoder();
+        const playerIdLen = dataView.getUint32(Uint32Array.BYTES_PER_ELEMENT, true);
+        const byteLength = dataView.getUint32(2*Uint32Array.BYTES_PER_ELEMENT, true);
+        const decodeMessage = textDecoder.decode(new Uint8Array(e.data, 3*Uint32Array.BYTES_PER_ELEMENT, byteLength));
+        const playerId = decodeMessage.slice(0, playerIdLen);
+        const chatMessage = decodeMessage.slice(playerIdLen);
+
+
+        const player = metaversefileApi.useRemotePlayer(playerId);
```
Comment (avaer) [inline wsrtc.js:188]: This belongs in `app`.

**Review by avaer — COMMENTED** [review]
The rest looks good.

File: wsrtc-server.mjs [hunk 3]
```diff
@@ -210,6 +210,13 @@ const bindServer = (server, {initialRoomState = null, initialRoomNames = []} = [
             })
             break;
           }
+          case MESSAGE.CHAT: {
+            room.players.forEach(player => {
+              if (player === localPlayer) return;
```
Comment (avaer) [inline wsrtc-server.mjs:215]: How about the server doesn't reflect messages back to the sayer?

Comment (lalalune) [reply wsrtc-server.mjs:215]: Not sure what you mean, currently it checks if player === localPlayer and doesn't reflect it.

Comment (avaer) [reply wsrtc-server.mjs:215]: Ah yup never mind, I thought this was frontend code!

**Review by avaer — APPROVED** [review]

**Pull request merged**